features/manage-position/Withdraw.tsx

```diff
@@ -440,4 +445,4 @@ const Deposit = () => {
   }
 };
 
-export default Deposit;
+export default Withdraw;
```

Comment:
unrelated to this PR but this was a typo we've had for a while

Reply:
Lol woops